doc/developers.rst

```diff
@@ -5,7 +5,7 @@ Developer Documentation
 We could use help with a number of ongoing issues. If you'd like to help,
 check the `issue tree <https://github.com/glm-tools/pyglmnet/issues>`_
 as well as our
-`requests for pull requests <https://glm-tools.github.io/requests.html>`_.
+`requests for pull requests <https://glm-tools.github.io/pyglmnet/requests.html>`_.
```

Comment:
Should we rename this file to be `Contributing.rst` or `Contributing.md` because that's where github expects it to be. Less surprises for new contributors.

Reply:
done